sdk/management/README.md

```diff
@@ -0,0 +1,221 @@
+# Azure management client library for Java
+
+The Azure Management Library for Java is a higher-level, object-oriented API for *managing* Azure resources,
+that is optimized for ease of use, succinctness and consistency.
+
+- [API reference documentation][docs]
+- [Samples][samples]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-management;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-management</artifactId>
+  <version>2.0.0</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Include optional packages
+
+[Azure Identity][azure_identity] provides Azure Active Directory token authentication support across the Azure SDK.
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-identity</artifactId>
```

Comment:
is azure-identity optional package? I can get the point our mgmt libs not depends on it. but from user's perspective, how to make it clear to them that they need azure-identity or TokenCredential before performing any mgmt operation. Maybe link to auth.md somewhere?

Reply:
It is optional. User can do their own OAUTH, without any lib. I think this is the exact reason that requires us not to include azure-identity module.